index.html

```diff
@@ -6,30 +6,32 @@
   </head>
   <body>
     <h1 class ="title quicksand">✨Self Care Center✨</h1>
-    <section class>
-      <h2 class ="title quicksand">Which type of message?</h2>
-    </section>
+    <h2 class ="quicksand sub-heading">Which type of message?</h2>
     <form class ="radio-box quicksand">
+      <br />
       <div class ="input-flex" >
         <div>
           <input type ="radio" id="choice1" name ="radio-button" value = "shallow-thought" checked>
-          <label for ="choice1" class ="text-italic">Shallow Thoughts</label>
+          <label for ="choice1" class ="text-italic">mantras</label>
         </div>
         <div>
           <input type ="radio" id ="choice2" name ="radio-button" value ="deep-thought" >
-          <label for ="choice2" class ="text-italic">Deep Thoughts</label>
+          <label for ="choice2" class ="text-italic">affirmations</label>
         </div>
       </div>
       <br />
-      <div>
-        <button class ="receive-message" type ="button">Get Knowledge</button>
-        <br />
-        <button class ="delete-message hidden" type ="button">Trash Knowledge</button>
+      <div class="message-buttons-flex">
+        <button class ="receive-message button quicksand" type ="button">Recieve Message</button>
+        <button class ="delete-message button hidden quicksand" type ="button">Delete Message</button>
       </div>
     </form>
     <section class ="image-box">
       <img class ="gong-image" src ="assets/meditate.svg" alt ="gong" />
-      <p class ="quicksand hidden" id ="message-display"></p>
+    <br />
+    <br />
+      <div class = image-box-flex>
```

Comment:
You don't need this div. If you go to dev tools and inspect this div it isn't the same size as the white box thus making it not perfectly center relative to the white image box. Try removing the div and adding `image-box-flex` to the class list above in the section element

Remove the breaks, and then play around with justify-content, align-items to try and properly center the text in the image box

Reply:
I knew this was a bad example of letting HTML do whatever I wanted.  I was getting some grief from the image and didn't have time to noodle around without finishing the rest of the project specs on time. Now that everything is in place, I will attempt to apply the correct method. Thank you for catching that!

Reply:
I am going to have to spend some more time with this in order to get it to display correctly. I will merge the original _sloppy_ HTML version, and attempt to make progress with it later. Thanks again for the feedback, It has been incredibly helpful!